content/source/docs/enterprise/private/preflight-installer.html.md

```diff
@@ -77,12 +77,38 @@ SELINUX=permissive
 
 ### Network Requirements
 
-1. Have the following ports available to the Linux instance:
-  * **22** - to access the instance via SSH from your computer. SSH access to the instance is required for administration and debugging.
-  * **8800** - to access the installer dashboard.
-  * **443** and **80** - to access the TFE app (both ports are needed; HTTP will redirect to HTTPS).
-  * **9870-9880 (inclusive)** - for internal communication on the host and its subnet; not publicly accessible.
-  * **23000-23100 (inclusive)** - for internal communication on the host and its subnet; not publicly accessible.
+The system that is running Terraform Enterprise must meet the following network
+requirements.
+
+#### Ingress
+
+Terraform Enterprise listens on the following ports for ingress connections:
+
+  * **22**: To access the instance via SSH from your computer. SSH access to the instance is required for administration and debugging.
+  * **80**: To access the Terraform Enterprise appplication via HTTP. This port redirects to port 443 for HTTPS.
+  * **443**: To access the Terraform Enterprise appplication via HTTPS.
+  * **8800**: To access the installer dashboard.
+  * **9870-9880 (inclusive)**: For internal communication on the host and its subnet; not publicly accessible.
+  * **23000-23100 (inclusive)**: For internal communication on the host and its subnet; not publicly accessible.
+
+#### Egress
+
+Terraform Enterprise connects to the following hostnames for egress connections:
+
+* registry.terraform.io
```

Comment:
If we're going to add this, we need to clarify when and why it's required.

Reply:
I agree. I was thinking about following the sub bullet syntax that's below it but I felt that would be a little cluttered.